mod_auth/controllers.py

```diff
@@ -255,8 +260,10 @@ def reset():
         user = User.query.filter_by(email=form.email.data).first()
         if user is not None:
             send_reset_email(user)
-        flash('If an account was linked to the provided email address, an email with reset instructions has been sent. '
-              'Please check your inbox.', 'success')
+            flash('An email with reset instructions has been sent. '
+                  'Please check your inbox.', 'success')
+        else:
```

Comment:
Can't merge while this is in here. For security reasons we don't want to divulge that an account is associated to a given email address.